http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -772,6 +772,7 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       raw"""galaxy.ingress.hosts[0]=${k8sProxyHost}""",
       raw"""galaxy.configs.galaxy\.yml.galaxy.single_user=${userEmail.value}""",
       raw"""galaxy.configs.galaxy\.yml.galaxy.admin_users=${userEmail.value}""",
+      raw"""galaxy.rbac.serviceAccount=${ksa.value}""",
       raw"""rbac.serviceAccount=${ksa.value}""",
```

Comment:
curious why there are two `serviceAccount` keys

Reply:
Good question -- it was in the latest `values` that Alex sent me, but I'm not sure the purpose of `galaxy.rbac.*`. I just pinged Alex for clarification.

Reply:
Alexandru Mahmoud  10:07 AM
the enabled should be false from the default, but yes setting both rbac.serviceAccount and galaxy.rbac.serviceAccount  is needed
10:07
(P.S. it's disabled so that the Galaxy chart doesn't deploy a new Role/RoleBinding, but the KSA name will be used by the pods) (edited)